[2025-02-12T16:03:02.993+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-25T17:00:00+00:00 [queued]>
[2025-02-12T16:03:03.006+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-25T17:00:00+00:00 [queued]>
[2025-02-12T16:03:03.007+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-02-12T16:03:03.030+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-01-25 17:00:00+00:00
[2025-02-12T16:03:03.041+0000] {standard_task_runner.py:57} INFO - Started process 2872 to run task
[2025-02-12T16:03:03.045+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-01-25T17:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_generator_dag.py', '--cfg-path', '/tmp/tmpektw6pqn']
[2025-02-12T16:03:03.046+0000] {standard_task_runner.py:85} INFO - Job 59: Subtask generate_customer_dim_data
[2025-02-12T16:03:03.106+0000] {task_command.py:415} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-25T17:00:00+00:00 [running]> on host 5879df32f101
[2025-02-12T16:03:03.211+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='hoyseiha@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-25T17:00:00+00:00'
[2025-02-12T16:03:03.222+0000] {logging_mixin.py:151} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-12T16:03:03.222+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-12T16:03:03.234+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, execution_date=20250125T170000, start_date=20250212T160302, end_date=20250212T160303
[2025-02-12T16:03:03.298+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-12T16:03:03.320+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
